### ISSUE TITLE
Access to archive.org fails unexpectedly (Easter egg)

### ISSUE THREAD

**Issue by LeadSongDog** [internetarchive/openlibrary#969]
User registration, or changing user's email or password requires access to archive.org, without which it cannot be done. (Group policies sometimes block archive.org as a gaming, porn, or warez site.) There is no hint to the user that this off-site (non-OL) access is happening, so the failure looks like something more sinister has happened.

Possible fixes could be a limited-scope proxy, register-by-email, or at least some warning text.

Tangential to #918

**Comment by LeadSongDog** [comment]
Tangential to #917

**Comment by guyjeangilles** [comment]
@LeadSongDog Is this still an issue? It seems the merging of IA and OL authentication has occurred recently. @mekarpeles Assigning you to reconcile this issue.

**Comment by LeadSongDog** [comment]
@guyjeangilles Yes, the links on the Settings page at https://openlibrary.org/account still go offsite to https://archive.org/account/index.php?settings=1 without warning the user they are about to do so. At a minimum there should be a heads up, but the better answer would be to make it work without resorting to forcing the users direct access to archive.org : much of OL's functionality works just fine without that access.

**Comment by mekarpeles** [comment]
Closing as not planned
